go.mod

```diff
@@ -12,3 +12,12 @@ require (
 	github.com/tj/go-progress v0.0.0-20180508172012-fadc638a53dd
 	gopkg.in/alecthomas/kingpin.v2 v2.2.6
 )
+
+require (
+	github.com/buger/goterm v0.0.0-20181115115552-c206103e1f37 // indirect
+	github.com/jmespath/go-jmespath v0.4.0 // indirect
+	github.com/mattn/go-colorable v0.1.8 // indirect
+	github.com/mattn/go-isatty v0.0.12 // indirect
+	github.com/pkg/errors v0.9.1 // indirect
+	golang.org/x/sys v0.0.0-20210423082822-04245dca01da // indirect
```

Comment:
@chenrui333 where are these deps being used?

Reply:
those are indirect dependencies (which are introduced with `go mod tidy`), I will merge `require` blocks together so that it would be better to understand.